### DISCUSSION TITLE
Language selector in the app

### DISCUSSION THREAD

**Discussion by introkun** [TreyWW/MyFinances#296] [Ideas]
As of now, UI language is selected automatically based on the browser language.
Let's add the ability to select language for the user.
#295 introduced translation ability and added User.language. But we should introduce UI for this to be able to override the browser's settings.

**Comment by TreyWW** [comment]
I'm really hesitant to add this mainly due to the fact it'll be hard to maintain. If we add new features, we'd need to get language translations for all text. And if some parts dont have full language support, it'll just get messy.
Would letting the browser take care of translations not be enough? I don't speak any other languages so I'm not sure how accurate they are.
